src/components/AddBookForm.js

```diff
@@ -211,18 +287,34 @@ const BookcoverContainer = styled.div`
   }
 
   label {
+    font-size: 1.2rem;
     background-color: #4a4453;
     color: #fff;
     display: flex;
     justify-content: center;
     align-items: center;
-    padding: 5px;
+    height: 40px;
     margin-left: 0;
     border-radius: 5px;
     font-weight: 600;
-    width: 80%;
+    width: 100%;
     text-transform: uppercase;
     box-shadow: var(--box-shadow);
     -webkit-box-shadow: var(--box-shadow);
   }
 `
+
+const ReadingStatus = styled.div`
+  display: flex;
+  justify-content: space-between;
+  height: 200px;
+
+  button {
+    background-color: #4a4453;
+    height: 40px;
+  }
+
+  button:focus {
+    height: 40px;
+  }
```

Comment:
Not completly sure here, but I think you can remove this one, because you already defined the hight. That said it could change on focus so you might need it.

Reply:
You guessed right, I have a general focus style for buttons in my GlobalStyles, but here I don't want that the height changes if focused. 😅